src/renderer/sw_engine/tvgSwRenderer.h

```diff
@@ -65,6 +65,7 @@ class SwRenderer : public RenderMethod
     void prepare(RenderEffect* effect, const Matrix& transform) override;
     bool region(RenderEffect* effect) override;
     bool render(RenderCompositor* cmp, const RenderEffect* effect, bool direct) override;
+    void dispose(RenderEffect* effect) override;
```

Comment:
Please leave as empty method here since software wouldn't use this dispose() method.
```
void dispose(TVG_UNUSED RenderEffect* effect) override {}
```